pkg/substitution/substitution.go

```diff
@@ -39,6 +40,51 @@ func setCloudControllerImage(config config.OperatorConfig, p corev1.PodSpec) cor
 	return updatedPod
 }
 
+// setProxySettings substitutes controller containers in provided pod specs with cluster wide proxy settings
+func setProxySettings(config config.OperatorConfig, p corev1.PodSpec) corev1.PodSpec {
+	clusterProxyEnvVars := getProxyArgs(config.ClusterProxy)
+	if len(clusterProxyEnvVars) == 0 {
+		return p
+	}
+
+	updatedPod := *p.DeepCopy()
+	for i, container := range p.Containers {
+		klog.Infof("Substituting proxy settings for container %q with %+v", container.Name, clusterProxyEnvVars)
```

Comment:
Do we definitely want to log this information? Perhaps the specific details could go at a higher log level?